03_launch_mgmt_cluster.sh

```diff
@@ -10,7 +10,7 @@ source lib/network.sh
 
 if [ "${EPHEMERAL_CLUSTER}" == "minikube" ]; then
   IRONIC_HOST="${CLUSTER_URL_HOST}"
-  export IRONIC_HOST_IP="${CLUSTER_PROVISIONING_IP}"
+  export IRONIC_HOST_IP="${PROVISIONING_IP}"
```

Comment:
Why do we need duplicated IRONIC_HOST_IP exported in the condition, maybe we can just export it before the condition?

Reply:
This is for testing only for now. The idea is that we want to have a single IP,  that's why there is a duplication.

Reply:
I don't get the reason @fmuyassarov  now line 13 and 16 are the same, you can take it out of the condition, as pointed out by @furkatgofurov7

Reply:
I am reverting the changes and do the variable change only for the ironic-configmap.